src/ngx_http_lua_headers.c

```diff
@@ -1415,4 +1467,19 @@ ngx_http_lua_ffi_get_resp_header(ngx_http_request_t *r,
 #endif /* NGX_LUA_NO_FFI_API */
 
 
+#if defined(nginx_version) && nginx_version >= 1011011
+void
+ngx_http_lua_ngx_raw_header_cleanup(void *data)
+{
+    ngx_http_lua_main_conf_t  *lmcf;
+
+    lmcf = (ngx_http_lua_main_conf_t *) data;
+
+    if (lmcf->busy_bufs_ptrs) {
+        ngx_free(lmcf->busy_bufs_ptrs);
```

Comment:
It's considered good practice to always clear out `lmcf->busy_bufs_ptrs` to avoid any chance of double frees.